data/trainers/dvs.asm

```diff
@@ -1,4 +1,6 @@
 TrainerClassDVs:
+; BUG: RIVAL2 has lower DVs than RIVAL1 (see docs/bugs_and_glitches.md)
```

Comment:
This can go right above the `RIVAL2` DV line. It's particularly convenient for lines like these to put them right next to where they're needed, to avoid this blank line between the label and the table contents.